www/caddy/pkg-descr

```diff
@@ -25,6 +25,10 @@ DOC: https://docs.opnsense.org/manual/how-tos/caddy.html
 Plugin Changelog
 ================
 
+1.6.2
+
+* Add: Authentik as authentication provider. [contributed by @Tim-Sc] 
```

Comment:
```suggestion
* Add Authentik as authentication provider (contributed by Tim-Sc)
```

just a little style and github handles are meaningless when showing this in the GUI